# Define a new message type called 'ProcessInfo'

float32[] process_usage_list
string[] process_name_list